attributes/default.rb

```diff
@@ -2,6 +2,9 @@
 default['packagecloud']['gpg_key_path'] = "/gpg.key"
 default['packagecloud']['hostname'] = nil
 
+default['packagecloud']['_stored_hostname'] = nil
+default['packagecloud']['override_hostname'] = nil
```

Comment:
we're open to renaming them -- we'd hoped that the `_` in `_stored_hostname` would indicate that it was an attribute for internal use for the cookbook.

perhaps something like:  `default['packagecloud']['internal'][...]` is more appropriate?